Gemfile

```diff
@@ -12,6 +12,8 @@ gem 'robot-controller' # robot code
 gem 'honeybadger' # for error reporting / tracking / notifications
 gem 'text-table' # to generate tables for StatsReporter
 gem 'whenever' # manage cron for robots and monitoring
+gem 'faraday' # for ReST calls
```

Comment:
if you wanted to get specific, it could say `# for ReST calls to Preservation Catalog` but not a biggie.